context/app/static/js/components/workspaces/utils.js

```diff
@@ -72,7 +75,10 @@ function mergeJobsIntoWorkspaces(jobs, workspaces) {
 
   const wsIdToJobs = {};
   jobs.forEach((job) => {
-    const { workspace_id } = job;
+    const { status, workspace_id } = job;
+    if (status === 'complete') {
```

Comment:
Are there other job statuses that will not block workspace deletion? For example, if it errored out?

Reply:
failed would be the other job status here, pending and running both block workspace deletion.